lib/utils.js

```diff
@@ -5,9 +5,9 @@
  *
  * @param filename - name of a file basename.ext
  */
-function basename (filename) {
-  let dotPosition = filename.lastIndexOf('.')
-  return dotPosition === -1 ? filename : filename.substring(0, dotPosition)
+function basename (filename, filetype) {
```

Comment:
Das ist eigentlich unnötig, da entweder die File extension sowas wie .png/.jpg/.docx ist --- dann funktioniert abschneiden beim letzten Punkt --- oder die Datei hat einen Punkt im Namen aber soll keine Extension haben this.is.a.bad.filename. Für den letzteren Fall kannst du ja aber stripExtension = false setzen und alles ist gut.